docs/src/man/getting_started.md

```diff
@@ -58,6 +59,13 @@ julia> df.A
  3
  4
 
+julia> df."A"
```

Comment:
I will mark it as resolved, as this is essentially the point of this PR to allow it (and I assume since @StefanKarpinski first proposed to accept `df."A"` it will keep getting supported by Julia parser :smile:).

Essentially the current recommendation was to write `df.var"some variable with spaces"` so we are just avoiding writing `var` in this case.

Indeed my initial fear was that allowing strings and `Symbol`s can lead to confusion but I think it is just a matter of documentation and tutorials.